### PR TITLE
Dark UI optimized by Tara

### PR DESCRIPTION
**Pull request by TaraRostami**
Dark UI optimized by Tara

### CODE REVIEW

**Comment by MHSanaei** [comment]
Hi @TaraRostami ,

Thank you for taking the time to work on this and contribute to the project. I noticed that there are a few issues with your pull request and design. Could you please take a look at them and make the necessary changes? If you need any help, let me know and I'll be happy to assist you.

Looking forward to seeing the updated version.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
-[![](https://img.shields.io/github/v/release/mhsanaei/3x-ui.svg)](https://github.com/MHSanaei/3x-ui/releases)
-[![](https://img.shields.io/github/actions/workflow/status/mhsanaei/3x-ui/release.yml.svg)](#)
-[![GO Version](https://img.shields.io/github/go-mod/go-version/mhsanaei/3x-ui.svg)](#)
-[![Downloads](https://img.shields.io/github/downloads/mhsanaei/3x-ui/total.svg)](#)
+[![](https://img.shields.io/github/v/release/TaraRostami/3x-ui.svg)](https://github.com/TaraRostami/Tara/releases)
+[![](https://img.shields.io/github/actions/workflow/status/TaraRostami/Tara/release.yml.svg)](#)
+[![GO Version](https://img.shields.io/github/go-mod/go-version/TaraRostami/3x-ui.svg)](#)
+[![Downloads](https://img.shields.io/github/downloads/TaraRostami/Tara/total.svg)](#)
 [![License](https://img.shields.io/badge/license-GPL%20V3-blue.svg?longCache=true)](https://www.gnu.org/licenses/gpl-3.0.en.html)
```
Comment (MHSanaei) [inline README.md:7]: fix this

File: README.md [hunk 2]
```diff
@@ -15,15 +15,15 @@ xray panel supporting multi-protocol, **Multi-lang (English,Farsi,Chinese)**
 # Install & Upgrade
 
 ```
-bash <(curl -Ls https://raw.githubusercontent.com/mhsanaei/3x-ui/master/install.sh)
+bash <(curl -Ls https://raw.githubusercontent.com/TaraRostami/Tara/master/install.sh)
```
Comment (MHSanaei) [inline README.md:18]: +

File: README.md [hunk 3]
```diff
@@ -15,15 +15,15 @@ xray panel supporting multi-protocol, **Multi-lang (English,Farsi,Chinese)**
...
 ```
 
 ## Install custom version
 
 To install your desired version you can add the version to the end of install command. Example for ver `v1.3.3`:
 
 ```
-bash <(curl -Ls https://raw.githubusercontent.com/mhsanaei/3x-ui/master/install.sh) v1.3.3
+bash <(curl -Ls https://raw.githubusercontent.com/TaraRostami/Tara/master/install.sh) v1.3.3
```
Comment (MHSanaei) [inline README.md:26]: +

File: install.sh [hunk 4]
```diff
@@ -108,20 +108,20 @@ install_x-ui() {
     cd /usr/local/
 
     if [ $# == 0 ]; then
-        last_version=$(curl -Ls "https://api.github.com/repos/MHSanaei/3x-ui/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+        last_version=$(curl -Ls "https://api.github.com/repos/TaraRostami/Tara/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```
Comment (MHSanaei) [inline install.sh:111]: and all your tag name here

**Comment by TaraRostami** [comment]
Hello, all previous problems have been solved

Re-install the panel version 1.3.5 and view the UI

Only the following problem is present in the white theme
which can be easily solved, you can fix it yourself
Screenshot 2023-05-08 004409

**Pull request closed without merging**